seed.sh

```diff
@@ -0,0 +1,33 @@
+echo "Username: $1"
+echo "Password: $2"
+echo "DBName: $3" 
+
+psql -U $1 $3 << EOF
+
+insert into users (name,email,password,is_admin) values ('Jan Doe','jandoe@gmail.com','12345',true);
+insert into users (name,email,password,is_admin) values ('Alisa Ray','alisaray@gmail.com','hello',false);
+insert into users (name,email,password,is_admin) values ('Tom Walters','tomwalters@gmail.com','tom123',false);
+insert into users (name,email,password,is_admin) values ('Alice Stephen','alicestep@gmail.com','alice776',true);
+
+insert into assets (id,status,category,purchase_at,purchase_cost,name,specifications) values ('ffb4b1a4-7bf5-11eb-9439-0242ac130002','active','Laptop','01/07/2017',50000,'Dell Latitude E5550','{"RAM":"4GB","HDD":"500GB","Generation":"i5"}');
+insert into assets (id,status,category,purchase_at,purchase_cost,name,specifications) values ('ffb4b488-7bf5-11eb-9439-0242ac130002','undermaintenance','Laptop','01/10/2017',40000,'IBM Thinkpad','{"RAM":"4GB","HDD":"500GB","Generation":"i5"}');
```

Comment:
do we have undermaintenance status?

Reply:
Yes we have defined status to take undermaintenance value